src/main/java/nl/tudelft/dnainator/ui/widgets/PropertyPane.java

```diff
@@ -0,0 +1,135 @@
+package nl.tudelft.dnainator.ui.widgets;
+
+import java.io.IOException;
+import java.util.List;
+
+import nl.tudelft.dnainator.ui.widgets.dialogs.ExceptionDialog;
+import javafx.animation.Animation;
+import javafx.animation.Transition;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import javafx.util.Duration;
+
+/**
+ * A pane displaying several properties of the displayed
+ * DNA sequences.
+ * <p>
+ * The pane is animated with slide-in and slide-out animations. The assumption
+ * is made (and enforced) that the pane starts being visible.
+ * </p>
+ */
+public class PropertyPane extends VBox {
+	private static final String FXML = "/ui/fxml/propertypane.fxml";
+	private static final int DURATION = 250;
+	private static final int WIDTH = 300;
+	@FXML private Label label;
+	@FXML private VBox vbox;
+	private Animation show;
+	private Animation hide;
+
+	/**
+	 * Creates a new property pane.
+	 */
+	public PropertyPane() {
+		loadFXML();
+
+		this.setPrefWidth(WIDTH);
+		this.setVisible(true);
+		this.setMinWidth(0);
+		setupHideAnimation(WIDTH, DURATION);
+		setupShowAnimation(WIDTH, DURATION);
+
+		getStyleClass().add("property-pane");
+		label.setId("properties-label");
+	}
+
+	private void loadFXML() {
+		FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource(FXML));
+		fxmlLoader.setRoot(this);
+		fxmlLoader.setController(this); // Necessary for the FXML injection.
+
+		try {
+			fxmlLoader.load();
+		} catch (IOException e) {
+			new ExceptionDialog(null, e, "Can not load the PropertyPane!");
+		}
+	}
+
+	private void setupHideAnimation(double expandedWidth, int duration) {
+		hide = new Transition() {
+			{ setCycleDuration(Duration.millis(duration)); }
```

Comment:
I wonder why check style doesn't whine about this, does it need to be a oneliner?